particle/pdgid/pdgid.py

```diff
@@ -19,14 +19,20 @@ class PDGID(int):
     >>> PDGID(11).is_lepton
     True
     """
-    __slots__ = ()
+    __slots__ = () # Keep PDGID a slots based class
 
     def __repr__(self):
         return "<PDGID: {:d}{:s}>".format(int(self),'' if self.is_valid else ' (is_valid==False)')
 
     def __str__(self):
         return repr(self)
 
+    def __neg__(self):
+        return self.__class__(-int(self))
+
+    def __invert__(self):
```

Comment:
Why not just return `__neg__` so that it immediately jumps to the eye that the 2 methods are set to do the same?

Reply:
Yes, I should. I was thinking about allowing discussions on this, since this `~` means something different on ints, so that `~int(x) != ~PDGID(x)`.